controller/job/rmdatahandler.go

```diff
@@ -0,0 +1,103 @@
+package job
+
+/*
+Copyright 2017 - 2020 Crunchy Data Solutions, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/crunchydata/postgres-operator/config"
+	"github.com/crunchydata/postgres-operator/kubeapi"
+	"github.com/crunchydata/postgres-operator/operator/pvc"
+	log "github.com/sirupsen/logrus"
+	apiv1 "k8s.io/api/batch/v1"
+)
+
+// handleRMDataUpdate is responsible for handling updates to rmdata jobs
+func (c *Controller) handleRMDataUpdate(job *apiv1.Job) error {
+
+	labels := job.GetObjectMeta().GetLabels()
+
+	// return if job wasn't successful
+	if !isJobSuccessful(job) {
+		log.Debugf("jobController onUpdate rmdata job %s was unsuccessful and will be ignored",
+			job.Name)
+		return nil
+	}
+
+	log.Debugf("jobController onUpdate rmdata job succeeded")
+
+	publishDeleteClusterComplete(labels[config.LABEL_PG_CLUSTER],
+		job.ObjectMeta.Labels[config.LABEL_PG_CLUSTER_IDENTIFIER],
+		job.ObjectMeta.Labels[config.LABEL_PGOUSER],
+		job.ObjectMeta.Namespace)
+
+	clusterName := labels[config.LABEL_PG_CLUSTER]
+
+	if err := kubeapi.DeleteJob(c.JobClientset, job.Name, job.Namespace); err != nil {
+		log.Error(err)
+	}
+
+	MAX_TRIES := 10
+	DURATION := 5
```

Comment:
I know this is "old code" but I think we should make these constants for umpteen reasons

Reply:
Went ahead and moved these to `consts` that are camelCased:
```
const (
	deleteRMDataJobMaxTries = 10
	deleteRMDataJobDuration = 5
)
```